db/projects/ardor/project.yaml

```diff
@@ -0,0 +1,71 @@
+name: Ardor
+start_date: '2018-01-01'
+description: 'Ardor is a platform securing various transaction sidechains.'
+web:
+  - 'https://www.ardorplatform.org/'
+ledger:
```

Comment:
It's not ledger, but **ledgers**
ledgers:

Reply:
fixed